### ISSUE TITLE
[+] ALT Repo: https://github.com/Azathothas/Toolpacks

### ISSUE THREAD

**Issue by Azathothas** [andrew-d/static-binaries#51]
Hello, first, I would like to apologize for shilling my own project here.
Please feel free to delete this issue if it's not acceptable.

Hopefully, this serves as a potential replacement/alternative to people.

To everyone else:
> - It seems like this repo was abandoned a while ago: `Last commit 4 Years ago`
> - I have a similar, much more ambitious project at: https://github.com/Azathothas/Toolpacks
> - There's already over [`2000+` precompiled binaries](https://bin.ajam.dev/) available to download, and all of them also come with a `.upx` packed versions.
> - Check my project out, and help me (& others) by [contributing](https://github.com/Azathothas/Toolpacks/tree/main/Docs#how-to-contribute)/[reporting Issues](https://github.com/Azathothas/Toolpacks/issues) there.

Thanks !

**Comment by xplshn** [comment]
Yup, and its over 3k binaries now, our most mature package manager sits at https://github.com/xplshn/dbin
